traffic_monitor/cache/cache.go

```diff
@@ -306,6 +307,17 @@ func (handler Handler) Handle(id string, rdr io.Reader, format string, reqTime t
 		handler.resultChan <- result
 		return
 	}
+	if val, ok := miscStats["plugin.system_stats.timestamp_ms"]; ok {
+		valString := fmt.Sprintf("%s", val)
```

Comment:
If you're doing `%s` successfully, then `val` must be either a string or a `fmt.Stringer` - why not just use the string value instead of relying on reflective formatting?

Reply:
Typically we void `fmt.Sprintf` in Traffic Monitor anyway for performance reasons, since it uses reflection (see https://github.com/apache/trafficcontrol/pull/6159#issuecomment-920952811 for more info), so this should use `strconv.Itoa`